web/frontend/src/components/board/index.svelte

```diff
@@ -95,8 +95,8 @@ const writeAnalog = async () => {
 
 const readAnalog = async () => {
   try {
-    const value = await boardClient.readAnalogReader(analogPinName);
-    readAnalogMessage = `${analogPinName}: value is ${value}`;
+    const reading = await boardClient.readAnalogReader(analogPinName);
```

Comment:
Nit assuming we can destructure reading:

```suggestion
    const { value } = await boardClient.readAnalogReader(analogPinName);
```